core/src/bufr/BufrReader/BufrParser.cpp

```diff
@@ -92,6 +94,66 @@ namespace bufr {
         return exportedData;
     }
 
+    std::shared_ptr<DataContainer> BufrParser::parse(const eckit::mpi::Comm& comm)
+    {
+      auto msgsToParse = std::floor(file_.size() / comm.size());
+      size_t startOffset = comm.rank() * msgsToParse;
+
+      // Messages may not split evenly among tasks, so distribute the remaining messages
+      if (auto remander = file_.size() - comm.size() * msgsToParse)
```

Comment:
remainder

Reply:
@CoryMartin-NOAA I will fix the spelling error. The `if (auto remainder = ....` statement is just means that the if contents of the `if` statement will execute if the result `remainder` evaluates to something that is `true` (in this case not 0). Writing the if statement in this way limits the scope of `remainder` to the `if` block and avoids polluting the namespace with temporary variables.

Reply:
Ok got it, I was thinking of equals and not if non-zero. This makes sense.